checks/docker_push.go

```diff
@@ -35,9 +43,15 @@ func (c *DockerPushChecker) Check(extConfig external.Check) *pkg.CheckResult {
 	check := extConfig.(v1.DockerPushCheck)
 	start := time.Now()
 	ctx := context.Background()
+	namespace := check.GetNamespace()
+	var err error
+	check.Auth, err = GetAuthValues(check.Auth, c.kommons, namespace)
```

Comment:
We should create a new auth object, not overwrite the check as the credentials may then leak back into a patch etc..

Reply:
Changed